integration_tests/tests/end_to_end.rs

```diff
@@ -532,3 +534,111 @@ async fn test_kafka() {
     })
     .await
 }
+
+#[test_case(
+    TestData {
+        station_id: 20001,
+        params: &[Param::new("TA"), Param::new("TGX")],
+        start_time: Utc::now().duration_trunc(TimeDelta::hours(1)).unwrap()
+            - Duration::hours(11),
+        period: Duration::hours(1),
+        type_id: 501,
+        len: 12,
+    }; "Scalar params")
+]
+#[tokio::test]
+async fn test_rove_connector(ts: TestData<'_>) {
+    e2e_test_wrapper(async {
+        let client = reqwest::Client::new();
+
+        let manager =
+            PostgresConnectionManager::new_from_stringlike(CONNECT_STRING, NoTls).unwrap();
+        let pool = bb8::Pool::builder().build(manager).await.unwrap();
+        let connector = rove_connector::Connector { pool };
+
+        let ingestor_resp = ingest_data(&client, ts.obsinn_message()).await;
+        assert_eq!(ingestor_resp.res, 0);
+
+        let resolution = "PT1H";
+        for param in ts.params {
+            let url = format!(
+                "http://localhost:3000/stations/{}/params/{}?time_resolution={}",
+                ts.station_id, param.id, resolution
+            );
+            let resp = reqwest::get(url).await.unwrap();
+
+            let json: TimeseriesResp = resp.json().await.unwrap();
+
+            let Timeseries::Regular(series) = &json.tseries[0] else {
+                panic!("Expected regular timeseries")
+            };
+
+            // feels kinda silly we had to use the API just to get the ts_id, but what can you do?
+            let ts_id = series.header.ts_id.to_string();
```

Comment:
You should be able to cheat with this, since `tsid`s are sequential and each test starts with a fresh DB.
Anyway why did we decide to have a String inside SpaceSpec::One? This should be different from the `tag` in `Timeseries`?

```suggestion
        for (i, param) in ts.params.iter().enumerate() {
            let ts_id = format!("{}", i + 1);
```

Reply:
> You should be able to cheat with this, since tsids are sequential and each test starts with a fresh DB.

Apparently not, I just tried this and it failed. The API reports ts_id 14 and 15 it seems. How are we clearing the DB, if we just empty the rows and don't reinit the table, that could explain the SERIAL values not being reused.

Reply:
> Anyway why did we decide to have a String inside SpaceSpec::One

SpaceSpec::One means we're asking for one timeseries, but we still need to know which timeseries, which the string specifies. It's a string because other data sources might use different types for their specifiers, e.g alphanumerics.

Reply:
> > You should be able to cheat with this, since tsids are sequential and each test starts with a fresh DB.
> 
> Apparently not, I just tried this and it failed. The API reports ts_id 14 and 15 it seems. How are we clearing the DB, if we just empty the rows and don't reinit the table, that could explain the SERIAL values not being reused.

Oh yeah, I was only testing the test you implemented :facepalm: But there's a TODO in the cleaup function, that's why it doesn't work right now I think

Reply:
Alright, what do you want to do to about this? Go fix the TODO now? Leave another TODO here?